test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -246,15 +245,12 @@ protected Layout getManagementTestLayout()
         bootstrapAllServers(l);
         corfuRuntime = getRuntime(l).connect();
 
-        // Waiting for management servers to send the bootstrap sequencer request and be ready
-        // to detect failures to speed up test time.
-        CFUtils.within(
-                CompletableFuture.allOf(
-                        getManagementServer(SERVERS.PORT_0).getManagementAgent().getRecoveryBarrierFuture(),
-                        getManagementServer(SERVERS.PORT_1).getManagementAgent().getRecoveryBarrierFuture(),
-                        getManagementServer(SERVERS.PORT_2).getManagementAgent().getRecoveryBarrierFuture()),
-                PARAMETERS.TIMEOUT_NORMAL
-        ).join();
+        // Waiting for sequencer to be bootstrapped
+        while(true) {
```

Comment:
Can we abstract this to a method?
Also instead of having a while loop lets have a for loop with `PARAMETERS.NUM_ITERATIONS_MODERATE` or so retries and sleep after checking for a `SHORT` time period. You can `fail()` if you exhaust the retries.

Reply:
Done